test/test_datapipe.py

```diff
@@ -1433,6 +1450,20 @@ def _mul_filter_fn(a, b):
         input_col_2_dp = tuple_input_ds.filter(_mul_filter_fn, input_col=[0, 2])
         self.assertEqual(list(input_col_2_dp), [(d - 1, d, d + 1) for d in range(5)])
 
+        # invalid input col
+        with self.assertRaises(TypeError):
+            tuple_input_ds.filter(_mul_filter_fn, input_col=0)
+
+        p_mul_filter_fn = partial(_mul_filter_fn, b=1)
+        out = tuple_input_ds.filter(p_mul_filter_fn, input_col=0)
+        self.assertEqual(list(out), [(d - 1, d, d+1) for d in range(10)])
```

Comment:
lint error

Reply:
Fixed in https://github.com/pytorch/pytorch/pull/79344/commits/2ca43e8310ed97c1e2439c37bf85a329e721e469